### PR TITLE
Lemuel Profile

### PR DESCRIPTION
**Pull request by rhymiz**
Check my shit. Please.

---

This change is [Review on Reviewable](https://reviewable.io/reviews/nw0428/sibspring2016/4)

### CODE REVIEW

**Comment by nw0428** [comment]
Add a link from index to your page.

File: css/lemuel/styles.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    font-family: 'Josefin Sans', sans-serif;
+    height: 100%;
+}
+
+.social-icon {
+    width: 35px;
+    height: auto;
+}
+
+.content-hidden {
```
Comment (nw0428) [inline css/lemuel/styles.css]: I think there should be two spaces before the {

**Comment by melissa0831** [comment]
the shit lgtm

**Comment by JQuezada0** [comment]
lgtm

**Comment by jdorancy** [comment]
Looks good to me.

File: css/lemuel/styles.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+* {
+    border-radius: 0 !important;
```
Comment (nw0428) [inline css/lemuel/styles.css:2]: Is !important really necessary? Why don't you want anything to override this?
What does !important do?

Comment (rhymiz) [inline css/lemuel/styles.css:2]: I'm overriding Boostrap's default border radius values. 
It's just a lazy approach to removing the rounded corners. 

using the below values won't work, hence the reason for using important to force the override.
- {
      border-radius: 0;
  }

**Pull request merged**